packages/core-components/src/components/EmptyState/EmptyState.tsx

```diff
@@ -46,7 +46,13 @@ type Props = {
   action?: JSX.Element;
 };
 
-/** @public */
+/**
+ * Various placeholder views for empty state pages
+ *
+ * @public
+ * @remarks
```

Comment:
You should probably remove all of these `@remarks` when there are no remarks - it marks the start of a longer section of the doc comment, where a longer-form description should be, often in several paragraphs.